sys/docker/alpine-nginx/start.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+# vim:sw=4:ts=4:et
```

Comment:
not needed: app specific, already handled by editorconfig ;)

Reply:
fixed